src/components/Modals/SetAclPermissionsModal.jsx

```diff
@@ -107,8 +123,8 @@ const SetAclPermissionsModal = ({ showModal, setShowModal, dataset }) => {
           dataset.modalType === 'container' ? 'Share All Documents' : `Share ${dataset.docName}`
         }
       >
-        <form onSubmit={handleAclPermission} autoComplete="off">
-          <FormControl required fullWidth sx={{ marginBottom: '1rem' }}>
+        <form onSubmit={handleAclPermission} autoComplete="off" style={{ width: '100%' }}>
+          <FormControl required fullWidth sx={{ maxWidth: '75vdw', marginBottom: '1rem' }}>
```

Comment:
Same unit here, think it should be `dvw` instead of `vdw`.

Reply:
Ah, good catch